kotlin/samples/todo-android/app/src/main/res/layout/master_detail_single.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Out of scope, but something to think about -- can we find a general way to get rid of this `FrameLayout` and just use the `WorkflowViewStub` directly?